tests/spread/plugins/v2/snaps/test-snapcraft-fake-ros2-package-core20/README.md

```diff
@@ -0,0 +1,6 @@
+# What is this, and why isn't it used?
```

Comment:
Is the intention that this package is maintained from here going forward? As in, this is the source tree to publish a new test-snapcraft-fake-ros2-package-core20?

Reply:
That's correct; much like it's ROS 1 counterpart [test-snapcraft-fake-ros-package-core20](https://github.com/snapcore/snapcraft/blob/5060014453a8c549aaa79de92a61e28efd0bf14d/tests/spread/plugins/v2/snaps/test-snapcraft-fake-ros-package-core20/README.md).